e2e/tests/stepDefinitions/booking.ts

```diff
@@ -0,0 +1,38 @@
+import { Given, Then } from '@badeball/cypress-cucumber-preprocessor'
+
+import {
+  BookingNewPage,
+  BookingConfirmationPage,
+  BookingFindPage,
+  PremisesShowPage,
+} from '../../../cypress_shared/pages/manage'
+
+import bookingFactory from '../../../server/testutils/factories/booking'
+import personFactory from '../../../server/testutils/factories/person'
+import keyWorkerFactory from '../../../server/testutils/factories/keyWorker'
+
+const keyWorker = keyWorkerFactory.build({ name: Cypress.env('keyworker_name') })
+const person = personFactory.build({ name: Cypress.env('offender_name'), crn: Cypress.env('offender_crn') })
```

Comment:
I'm thinking maybe we can be more helpful if these expected environment variables aren't present, and raise a clear error early?

e.g.

```ts
const keyWorkerName = Cypress.env('keyworker_name')
expect(keyWorkerName, 'keyWorkerName was set').to.be.a('string').and.not.be.empty
```

I've made a PR (https://github.com/ministryofjustice/hmpps-approved-premises-ui/pull/208) for doing this to existing Cypress variables in `e2e/tests/stepDefinitions/manage.ts`